src/hooks/useTokenBalances.ts

```diff
@@ -68,10 +72,7 @@ export const useTokenBalances = (): UseTokenBalanceResult => {
   useEffect(() => {
     _getBalances(walletInfo)
       .then(balances => {
-        console.log(
-          '[useTokenBalances] Wallet balances',
-          balances ? balances.map(b => formatAmount(b.walletBalance)) : null,
-        )
+        log('[useTokenBalances] Wallet balances', balances ? balances.map(b => formatAmount(b.walletBalance)) : null)
```

Comment:
Use the `decimals` amount when formating otherwise this won't be consistent for all tokens